requirements/base.txt

```diff
@@ -16,9 +16,9 @@ python-novaclient>=7.1.2,<16
 tenacity>=6.2,<8
 
 # for hastexo_guacamole_client
-django<3.2
-channels>=2.4.0,<3
-asgiref~=3.2.0 # depends on our channels version
+django~=3.2.1
+channels>2.4.0,<=3.0.5
```

Comment:
I think this is functionally correct (there has been no Channels 2.x release since 2.4.0), but just to make it clearer you may want to set this to `channels>=3.0.0,<=3.0.5` or perhaps `channels~=3.0.0` which more clearly reflects that this depends on the Channels 3.x.x major release.

Reply:
Ack, done. I went with `channels>=3.0.0,<=3.0.5` because I think it's better if we bump the upper requirement for channels and asgiref consciously instead of allowing a newer version being pulled in once released without testing it before.